firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/GenerationConfig.kt

```diff
@@ -44,6 +44,10 @@ package com.google.firebase.vertexai.type
  * unique candidates. Setting `temperature` to 0 will always produce exactly one candidate
  * regardless of the `candidateCount`.
  *
+ * @property presencePenalty Positive penalties.
+ *
+ * @property frequencyPenalty Frequency penalties.
```

Comment:
Are these descriptions from the proto? It'd be nice to have more descriptive descriptions.

Reply:
Yeap, directly from the proto. Will work on improve them as a follow up